src/components/DocumentModals/ShowDocumentLocalModal.jsx

```diff
@@ -0,0 +1,54 @@
+// React Imports
+import React from 'react';
+// Material UI Imports
+import Button from '@mui/material/Button';
+import Dialog from '@mui/material/Dialog';
+import DialogActions from '@mui/material/DialogActions';
+import DialogContent from '@mui/material/DialogContent';
+import DialogTitle from '@mui/material/DialogTitle';
+import ClearIcon from '@mui/icons-material/Clear';
+
+/**
+ * ShowDocumentsModal Component - Popup modal that shows client documents if gotten
+ * explicit permission via ACL
+ *
+ * @memberof DocumentModals
+ * @name ShowDocumentLocalModal
+ */
+
+const ShowDocumentLocalModal = ({ showModal, setShowModal, fileBlobUrl }) => (
+  <Dialog
+    fullScreen
+    open={showModal}
+    aria-labelledby="dialog-title"
+    onClose={() => setShowModal(false)}
+  >
+    <DialogTitle id="dialog-tile">Documents</DialogTitle>
```

Comment:
should this be the file name or docType instead maybe?

Reply:
Just Documents